### PR TITLE
Add flag of Chile 

### PR DESCRIPTION
**Pull request by adavis46**
added flag of Chile

### CODE REVIEW

File: Flags/chile.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Chile | Flags</title>
+  <meta charset="utf-8" />
+  <style>
+    body {
+        padding: 20px;
```
Comment (ManrajGrover) [inline Flags/chile.html]: Please fix indentation and make it consistent.

File: Flags/chile.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      }
+      #chile {
+        width: 900px;
+        height: 600px;
```
Comment (ManrajGrover) [inline Flags/chile.html]: Nit: Indentation.

File: Flags/chile.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+       background: repeating-linear-gradient(180deg,
+            white, white 300px,
+            #d52b1e 300px, #d52b1e 600px);
```
Comment (ManrajGrover) [inline Flags/chile.html]: Nit: Indentation.

**Comment by ManrajGrover** [comment]
Hey! @adavis46 This is great! Please fix the indentation issues I have commented. Make them more consistent. :)

**Comment by ManrajGrover** [comment]
@adavis46 Please resolve merge conflicts too. :blush:

**Comment by adavis46** [comment]
Corrected merge conflicts

File: README.md [hunk 4]
```diff
@@ -49,9 +49,13 @@ Please go through the [Contributions](https://github.com/ManrajGrover/SingleDivP
 | [Bangladesh](https://github.com/ManrajGrover/SingleDivProject/blob/master/Flags/Bangladesh.html) | [ManrajGrover](https://github.com/ManrajGrover) | ![Bangladesh](https://github.com/ManrajGrover/SingleDivProject/blob/master/Assets/Bangladesh.png) |
 | [Belgium](https://github.com/ManrajGrover/SingleDivProject/blob/master/Flags/Belgium.html) | [AndyMathys](https://github.com/AndyMathys) | ![Belgium](https://github.com/ManrajGrover/SingleDivProject/blob/master/Assets/Belgium.png) |
 | [Bulgaria](https://github.com/ManrajGrover/SingleDivProject/blob/master/Flags/Bulgaria.html) | [ManrajGrover](https://github.com/ManrajGrover) | ![Bulgaria](https://github.com/ManrajGrover/SingleDivProject/blob/master/Assets/Bulgaria.png) |
+<<<<<<< HEAD
```
Comment (ManrajGrover) [inline README.md]: @adavis46 Please remove this line too.

File: README.md [hunk 5]
```diff
@@ -49,9 +49,13 @@ Please go through the [Contributions](https://github.com/ManrajGrover/SingleDivP
...
+| [Chile](https://github.com/ManrajGrover/SingleDivProject/blob/master/Flags/chile.html) | [adavis46](https://github.com/adavis46) | ![Chile](https://github.com/ManrajGrover/SingleDivProject/blob/master/Assets/chile.PNG) |
+=======
```
Comment (ManrajGrover) [inline README.md]: @adavis46 Same here

**Comment by ManrajGrover** [comment]
@adavis46 There are some merge conflicts now. :worried: Please solve them and I'll merge it. :smile:

**Comment by ManrajGrover** [comment]
@adavis46 Looks good to me. :smile: Merging this. Thank you for contributing! :smile:

**Pull request merged**